config/redis.yml

```diff
@@ -41,6 +41,9 @@ development: &defaults
   sm_store_mobile:
     namespace: sm-service-mobile
     each_ttl: 1200
+  mr_store:
+    namespace: mr-service
```

Comment:
Does this _need_ to be an acronym? I'm not familiar with how this namespace would present itself, but I'm just thinking if I'm digging through logs and see `mr-service`, I wouldn't know what it's referring to. So... I guess I'm asking if you could rename this to `medical-records-service`. BUT, if this is really don't matter and never shows itself, don't worry about changing it 😃

Reply:
I was following the naming conventions that were used for other MHV applications, e.g. sm, bb, rx -- but I agree that here it could probably have a better name. I'll change it!

Reply:
nice! I like it 👍